app/scripts/lib/app-start.js

```diff
@@ -652,22 +650,12 @@ function (
     },
 
     _selectStartPage: function () {
-      var self = this;
-      return p().then(function () {
-        if (self._window.location.pathname === '/cookies_disabled') {
-          // If the user is already at the cookies_disabled page, don't even
-          // attempt the cookie check or else a blank screen is rendered if
-          // cookies are actually disabled.
-          return;
-        }
-
-        return self._configLoader.areCookiesEnabled(false, self._window)
-            .then(function (areCookiesEnabled) {
-              if (! areCookiesEnabled) {
-                return 'cookies_disabled';
-              }
-            });
-      });
+      if (
```

Comment:
nit: formatting. We go somewhere in the middle of verbose and terse, depending on the repo. We are a bit more terse in this repo and often combine lines that other repos do not.

```js
if (this._window.location.pathname !== '/cookies_disabled' &&
   ! this._storage.isLocalStorageEnabled(this._window)) {
```